src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests_ListPatterns.cs

```diff
@@ -1341,7 +1322,7 @@ public static void Main()
 True
 True
 Length
-False
+True
```

Comment:
:memo: This is testing `length<0`

Reply:
Should I assume that this is essentially undefined behavior? If so, we should make a comment saying as such in the test.

Reply:
The spec may need to be clarified, but I think we'll call this out and make it defined behavior. `..` with no nested pattern always matches without any evaluations, but `.. <nested pattern>` will evaluate the slice.

Reply:
One more note on this: I think the naked `..` pattern won't even require a sliceable type. We do have a bullet in test plan to confirm this with LDM though: "confirm that [ .._ ] requires a sliceable type, whereas [ 1, .. ] doesn't."

Reply:
>Should I assume that this is essentially undefined behavior?

Yes and it's being called out in the spec. 

@jcouv  "undefined behavior" here refers to the length being negative.